german/the-black-hack.md

```diff
@@ -289,7 +289,7 @@ Diebe würfeln mit einem **Vorteil** bei **GES**-Proben um Schaden oder Effekte
 
 Diebe würfeln mit **Vorteil** wenn sie von hinten angreifen und verursachen dabei 2W6 / 2W4 + ihre Stufe Schaden.
 
-Diebe würfeln mit **Vorteil** bei Tätigkeiten, die mit Fingerfertigkeit, Klettern, Lauschen, Schleichen, Lesen von Schrift und Schlösserknacken zu tun haben.
+Diebe würfeln mit **Vorteil** bei Tätigkeiten, die mit Fingerfertigkeit, Klettern, Lauschen, Schleichen, dem Verstehen von Schrift und Schlösserknacken zu tun haben.
```

Comment:
Dann also mit dem Verstehen von Schriften am Ende der Aufzählung. (Das Verstehen ist in dem Satz falsch und nur "Schriften verstehen" klingt komisch auch wenn es einfacher formuliert ist.)